R/brmsfit-helpers.R

```diff
@@ -973,27 +979,72 @@ brmsfit_needs_refit <- function(fit, sdata = NULL, scode = NULL, data = NULL,
   if (!is.null(algorithm)) {
     if (algorithm != fit$algorithm) {
       if (!silent) {
-        message("Algorithm has changed from '", fit$algorithm,
-                "' to '", algorithm, "'.\n")
+        message(
+          "Algorithm has changed from '", fit$algorithm,
+          "' to '", algorithm, "'.\n"
+        )
       }
       refit <- TRUE
     }
   }
   refit
 }
 
-# read a brmsfit object from a file
-# @param file path to an rds file
-# @return a brmsfit object or NULL
-read_brmsfit <- function(file) {
-  file <- check_brmsfit_file(file)
+
+#' Determine Cache Folder for brms
+#'
+#' Checks if the provided file path includes a directory. If so, returns that directory.
+#' Otherwise, looks for a user‐defined `brms_cache_folder` option. If the option is not set,
```

Comment:
please name the option "brms.cache_folder" for consistency with other brms options.